src/index.js

```diff
@@ -1,23 +1,21 @@
 'use strict';
 
 const express = require('express');
-const db = require('./db');
-const router = require('./router');
+const path = require('path');
+const bodyParser = require('body-parser');
 const defaultErrorHandler = require('./middleware/default-error-handle');
-const app = express();
 
-//app.set('view engine', 'ejs');
-//app.use('/static', express.static('static'));
-// Add middleware to parse JSON
+const router = require('./router');
 
-app.use(express.urlencoded());
-app.use(express.json());
-app.use(defaultErrorHandler);
-// Add router as middleware
+
+const app = express();
+
+app.set('view engine', 'ejs');
+app.use('/static', express.static(path.resolve('./static/css')));
```

Comment:
Preferable if use static for the entire directory that would include subdirectories including css ie.

```js
  app.use('/static', express.static(path.resolve('./static')));
```